small-town/src/App.js

```diff
@@ -15,6 +16,17 @@ class App extends React.Component {
 		};
 	}
 
+	componentDidMount() {
+		axios
+			.get('http://localhost:3100/people')
+			.then((result) => {
+				this.setState({ people: result.data });
+			})
+			.catch((error) => {
+				console.log(error);
```

Comment:
Poti afisa si eroarea pe ecran in caz ca nu reuseste requestul. Sa ai in state error:null si  in catch - sa setezi cu setState